shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/dal/reset.xml

```diff
@@ -19,4 +19,15 @@
 <sql-cases>
     <sql-case id="reset_all" value="RESET ALL"  db-types="PostgreSQL,openGauss" />
     <sql-case id="reset_timezone" value="RESET timezone"  db-types="PostgreSQL,openGauss" />
+    <sql-case id="reset_master" value="RESET MASTER" db-types="MySQL"/>
+    <sql-case id="reset_slave" value="RESET SLAVE" db-types="MySQL"/>
+    <sql-case id="reset_master_slave" value="RESET MASTER, SLAVE" db-types="MySQL"/>
+    <sql-case id="reset_master_with_binlog" value="RESET MASTER TO 10" db-types="MySQL"/>
+    <sql-case id="reset_slave_with_all" value="RESET SLAVE ALL" db-types="MySQL"/>
+    <sql-case id="reset_slave_with_channel" value="RESET SLAVE FOR CHANNEL 'TEST_CHANNEL'" db-types="MySQL"/>
+    <sql-case id="reset_slave_with_all_channel" value="RESET SLAVE ALL FOR CHANNEL 'TEST_CHANNEL'" db-types="MySQL"/>
+
```

Comment:
@flycash Please remove this useless black line.

Reply:
done